src/FSharp.Core/list.fsi

```diff
@@ -1710,6 +1910,9 @@ module List =
     /// through the computation. Apply the function to the first two elements of the list.
     /// Then feed this result into the function along with the third element and so on. 
     /// Return the final result. If the input function is <c>f</c> and the elements are <c>i0...iN</c> then computes 
+    /// 
+    /// Time complexity: O(n) - where n is the list length
+    /// Space complexity: O(1)
```

Comment:
Here the added complexity information should be included at the end of the summary and not in the middle of a sentence